### PR TITLE
Fix mini player custom layout.

### PR DESCRIPTION
**Pull request by haibthe**
Fix FUZVideo custom layout.

### CODE REVIEW

File: uizacoresdk/src/main/res/layout/layout_stats_for_nerds.xml [hunk 1]
```diff
@@ -2,14 +2,12 @@
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    >
+    android:layout_height="wrap_content">
```
Comment (namnh-0652) [inline uizacoresdk/src/main/res/layout/layout_stats_for_nerds.xml]: Please revert this change if you don't really change it!

Comment (deathemperor) [reply uizacoresdk/src/main/res/layout/layout_stats_for_nerds.xml]: it's OK Nam, this changes make the view easier to read.

File: uizacoresdk/build.gradle [hunk 2]
```diff
@@ -18,7 +18,9 @@ android {
         //https://stackoverflow.com/questions/41098015/how-to-set-vectordrawable-as-an-image-for-imageview-programmatically
         //https://stackoverflow.com/questions/49014416/set-vector-drawable-to-imageview-in-xml
         //https://stackoverflow.com/questions/34417843/how-to-use-vector-drawables-in-android-api-lower-21
-        vectorDrawables.useSupportLibrary = true*/
+        */
+        vectorDrawables.useSupportLibrary = true
```
Comment (namnh-0652) [inline uizacoresdk/build.gradle]: Currently, the sdk still not use `vector`, So, should we enable this from application side?

**Comment by deathemperor** [comment]
closed in favour of https://github.com/uizaio/uiza-android-sdk-player/pull/171
